.github/dependabot.yml

```diff
@@ -2,16 +2,13 @@ version: 2
 updates:
   - package-ecosystem: "cargo"
     directory: "/src/rust/engine"
-    open-pull-requests-limit: 3
+    open-pull-requests-limit: 5
     rebase-strategy: auto
     schedule:
       interval: "weekly" 
-      day: "thursday"
-      time: "12:00"
+      day: "wednesday"
+      time: "03:00"
```

Comment:
so 3pam PST, which should make them ready for review by the time people wake up so they can be merged in the morning or at the end of the work day.